seedfarmer/commands/_bootstrap_commands.py

```diff
@@ -63,15 +66,21 @@ def write_template(template: Dict[Any, Any]) -> None:
 
 def bootstrap_toolchain_account(
     project_name: str,
-    principalARNs: List[str],
-    permissionsBoundaryARN: Optional[str] = None,
+    principal_arns: List[str],
+    permissions_boundary_arn: Optional[str] = None,
     synthesize: bool = False,
+    as_target: bool = False,
 ) -> Optional[Dict[Any, Any]]:
-    template = get_toolchain_template(project_name, principalARNs, permissionsBoundaryARN)
+    template = get_toolchain_template(project_name, principal_arns, permissions_boundary_arn)
     _logger.debug((json.dumps(template, indent=4)))
     if not synthesize:
-        # call the services to deploy
-        pass
+        deploy_template(template=template, stack_name=f"seedfarmer-{project_name}-toolchain-role")
+        if as_target:
+            bootstrap_target_account(
+                toolchain_account_id=get_account_id(),
```

Comment:
since get_account_id() can now take session, shouldn't we explicitly pass one in to ensure we're getting account_id of the toolchain account?

Reply:
I would agree...but in this case there are 2 mitigating issues:

1. We never generate a session.....codeseeder does that and does not accept a session....we should change that
2. this call is synchronous and right after the call to the toolchain account, so there is no oppty to change sessions 

I would prefer to change #1 and then generate a session (with profile support) and pass that into codeseeder (optional parameter